RecoTracker/IterativeTracking/python/iterativeTkConfig.py

```diff
@@ -116,7 +139,8 @@ def createEarlySequence(era, modDict):
     return seq
 
 def iterationAlgos(era):
-    return [_modulePrefix(i) for i in globals()["_iterations"+postfix(era)] + _iterations_muonSeeded]
+  if era == "trackingPhase2PU140": return [_modulePrefix(i) for i in globals()["_iterations"+postfix(era)] + _iterations_muonSeeded_trackingPhase2PU140]
+  return [_modulePrefix(i) for i in globals()["_iterations"+postfix(era)] + _iterations_muonSeeded]
```

Comment:
Something along the following would be more generic
```
    muonVariable = "_iterations_muonSeeded"+postfix(era)
    return [_modulePrefix(i) for i in globals()["_iterations"+postfix(era)] + globals().get(muonVariable, _iterations_muonSeeded)
```
Please keep the indentation at 4 spaces.